keanu-project/src/main/java/io/improbable/keanu/util/status/TimeComponent.java

```diff
@@ -0,0 +1,11 @@
+package io.improbable.keanu.util.status;
+
+import java.time.Duration;
+
+public abstract class TimeComponent implements StatusBarComponent {
+
+    protected String formatDuration(Duration duration) {
+        // Duration.toString() uses the ISO-8601. This strips the `PT` at the start off.
+        return duration.toString().substring(2);
```

Comment:
I think we can print a easier to read format. The "S" looks like a 5 for seconds and for a lot of cases I'm not sure you want a resolution in milliseconds (e.g. time remaining).

Reply:
New format looks so good!